MetriCam2.Controls/CameraExplorerControl.cs

```diff
@@ -117,13 +115,10 @@ public void RefreshAvailableCameras()
                 listViewItem.Name = cam.GetType().ToString();
                 listViewItem.Cam = cam;
 
-                Bitmap bmp = ScaleBitmap(cam.CameraIcon, IMAGE_SIZE_LARGE);
-                imageListLarge.Images.Add(cam.GetType().ToString(), (Bitmap)bmp.Clone());
-                imageListSmall.Images.Add(cam.GetType().ToString(), ScaleBitmap(cam.CameraIcon, IMAGE_SIZE_SMALL));
+                imageListSmall.Images.Add(cam.GetType().ToString(), cam.CameraIcon);
```

Comment:
This is a change to the original code.
The icon was added to both lists (small/large), twice (with type name, and with "_C" suffix).
The version with the _C suffix would be used for connected cameras, so that is the version which should get the overlay / connect indicator (once it's done).